api/python/cellxgene_census/src/cellxgene_census/_release_directory.py

```diff
@@ -137,38 +137,18 @@ def get_census_version_description(census_version: str) -> CensusVersionDescript
     return description
 
 
-def get_census_version_directory_of_lts_releases(
-    include_retracted: bool = False,
-) -> Dict[CensusVersionName, CensusVersionDescription]:
-    """
-    Get a view of the directory that only includes the long-term stable ("LTS") Census releases, optionally including
-    retracted releases.
-
-    Params:
-        include_retracted: Include retracted releases in the returned directory.
-
-    Returns:
-        A dictionary that contains release names and their corresponding release description, for LTS releases only.
-
-    Lifecycle:
-        maturing
-
-    See Also:
-        :func:`get_census_version_directory`: get directory of all Census releases currently available.
-        :func:`get_census_version_description`: get description by census_version.
-    """
-    release_flags = dict(lts=True) if include_retracted else dict(lts=True, retracted=False)
-    return get_census_version_directory(**release_flags)
-
-
 # TODO: Update example output and add example for use of release_flags
-def get_census_version_directory(**release_flags_kwargs: bool) -> Dict[CensusVersionName, CensusVersionDescription]:
+def get_census_version_directory(
+    lts: Optional[bool] = None, retracted: Optional[bool] = None
```

Comment:
I am wondering if an unspecified `retracted` flag should not default to `True`

Pros/cons:
* we really don't want people using retracted datasets, so defaulting `True` accomplishes it
* if somewhat _is_ using a dataset, which is retracted after they being using it, the API may start to fail in a surprising manner.  Arguably a good thing, as we _want_ them to notice this change in status.

Is there a use case or motivation for a default to False?